### PR TITLE
Fixed mail button

### PR DESCRIPTION
**Pull request by Nivet2006**
Hello @omkarpatke,
I am Nived.
I made small changes in your portfolio website.
The send mail button was not working but now I have fixed it
Please consider merging this pull request
for more info. you may mail me at minds0987@gmail.com
Thank you
Waiting for you to merge this pull request
@nived-dev

### CODE REVIEW

**Comment by Nivet2006** [comment]
https://deploy-preview-1--omkar-patke.netlify.app/
Click on the above link to view the change

**Comment by omkarborude** [comment]
Omkar merge this pull request !

**Comment by Nivet2006** [comment]
> Omkar merge this pull request !

Yes, Merge it!!

File: index.html [hunk 1]
```diff
@@ -119,11 +119,11 @@ <h2 class="footer-heading">LET'S CONNECT</h2>
             </li>
 
             <li class="footer-link">
-                <a href="omkarpatke1101@gmail.com" class="link-center" target="_blank"><img src="images/mail.svg" alt="email link"></a>
+                <a href="mailto:omkarpatke1101@gmail.com" class="link-center" target="_blank"><img src="images/mail.svg" alt="email link"></a>
```
Comment (Nivet2006) [inline index.html:122]: Hello,
@omkarpatke , I have just edited the mail button.
So, I request you to merge this pull request! #1
 
Thank you,
Waiting for you to pull this request!!
@nived-dev
coding@[minds0987](https://github.com/minds0987)
#Stay home #Stay safe

**Comment by Nivet2006** [comment]
Hello,
@omkarpatke , I have just edited the mail button.
So, I request you to merge this pull request! #1
 
Thank you,
Waiting for you to pull this request!!
@nived-dev
coding@[minds0987](https://github.com/minds0987)
#Stay home #Stay safe

**Pull request closed without merging**